io/xmlparser/CMakeLists.txt

```diff
@@ -31,6 +28,10 @@ ROOT_STANDARD_LIBRARY_PACKAGE(XMLParser
     Core
 )
 
+# not needed to propagate includes to rootcling
+target_include_directories(XMLParser PRIVATE ${LIBXML2_INCLUDE_DIR})
```

Comment:
Here next two lines could be more combined in single call of target_include_directories()